README.md

```diff
@@ -50,38 +50,14 @@ Welcome to Traffic Courts Online
 
 # Splunk
 
-The default `docker-compose.yaml` file contains a local splunk service. A custom configuration file, `./.docker/splunk-dev-config.yaml`
+The default `docker-compose.yaml` file does NOT contain Splunk configuration. To use/test with Splunk,
+you can include 
+
+A custom configuration file, `./.docker/splunk-dev-config.yaml`
 is used to adjust the default settings. A key setting is disabling the SSL on the HEC endpoint.
 
 https://splunk.github.io/docker-splunk/EXAMPLES.html#create-standalone-with-hec
 
```

Comment:
I would be nice to include in the README example uses of jaeger, seq, etc.
Maybe just 
```
docker-compose -f docker-compose.yml -f ./docker/docker-compose.seq.yml up -d
```

Reply:
I actually had, but forgot to save/commit it. Pushed changes in last commit.

Reply:
This looks much better. Thanks.